src/MoBi.Core/Events/Events.cs

```diff
@@ -20,8 +20,21 @@ public SimulationRunFinishedEvent(IMoBiSimulation simulation)
       }
    }
 
+   public class SimulationsRunCanceledEvent
```

Comment:
Should we rename this to be clear that it's plural. I missed it the first two times I read?
AllSimulationRunsCanceledEvent?